scripts/test_ports/rpath-test-binaries/project/main.cpp

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+extern const char* getTestString();
+
+int main()
+{
+    printf("%s\n", getTestString());
```

Comment:
```suggestion
    puts(getTestString());
```

Only if there need to be other edits :)